apix-impl/src/main/java/eu/xenit/apix/alfresco/search/LuceneNodeVisitor.java

```diff
@@ -26,6 +26,10 @@ public LuceneNodeVisitor() {
         termDictionary.put("category", "CATEGORY");
         termDictionary.put("text", "TEXT");
         termDictionary.put("all", "ALL");
+        termDictionary.put("isunset", "ISUNSET");
```

Comment:
I was trying to understand why there are **two** dictionaries like this, so I searched where _LuceneNodeVisitor_ is used. I can't find any usages!
Why did you add it here? Maybe I'm missing something?

Reply:
I think is was added for completeness.
When Lucene is not supported anymore by the search backend in Alfresco 5.x - 6.x then it can be removed.
That clean up should have been part of the removal of support for Alfresco 4.2. Unless impact is low the LuceneNodeVisitor can be removed in this ticket otherwise I would suggest to create a ticket to clean up unused code

Reply:
hmm, I added it because I was looking where the other keywords were used and added the new ones to it. Since LuceneNodeVisitor is only used in tests I don't have to add the new keywords there. I will revert these changes.

I think the LuceneNodeVisitor was added when we were still supporting Alfresco 4.1 and 4.2 but since we don't do that anymore we also don't need that visitor anymore.

Reply:
I will create a ticket for the removal of the LuceneNodeVisitor and its related tests.